### ISSUE TITLE
GitLab open source program

### ISSUE THREAD

**Issue by ben-schwen** [Rdatatable/data.table#6242]
We could use the already available beta runner for macos if we apply for the Gitlab open source program.

_Originally posted by @ben-schwen in https://github.com/Rdatatable/data.table/issues/3326#issuecomment-2200975001_
            
I just got the notification that we are accepted to the open source program.

To finish we need to do the following:
- [ ] estimate the number of users for our GitLab repo (currently we are Michael, Jan, Matt and me), but I would add at least Toby.
- [ ] set my role to owner at the GitLab repo @jangorecki

We also need to renew the application every year, so we might want to introduce a role for that. For now, I'm happy to handle it.

**Comment by MichaelChirico** [comment]
Great news!

**Comment by tdhock** [comment]
cool thanks for the invite 
I almost never go on gitlab but I still have access to user id tdhock

**Comment by jangorecki** [comment]
By the end of the July I can get into gitlab and set an owner so this issue has to wait a bit.

**Comment by tdhock** [comment]
related: Aren't dev binaries built on gitlab? 
Is there any documentation for that anywhere? can people who know about how that works please write docs on this wiki page https://github.com/Rdatatable/data.table/wiki/Binary-managers 
Also I created a team https://github.com/orgs/Rdatatable/teams/binary-managers and I recommend adding people who know about that to this team for easy mentioning `@Rdatatable/binary-managers`

**Comment by ben-schwen** [comment]
> related: Aren't dev binaries built on gitlab? Is there any documentation for that anywhere? can people who know about how that works please write docs on this wiki page https://github.com/Rdatatable/data.table/wiki/Binary-managers Also I created a team https://github.com/orgs/Rdatatable/teams/binary-managers and I recommend adding people who know about that to this team for easy mentioning `@Rdatatable/binary-managers`

Yes. AFAIK the only documentation is the code in `.gitlab-ci.yml` and our issues here. Basially, the binaries get built before, copied in the integration step and published in the publish step. Afterwards they can be downloaded with the appropriate filepath e.g. for [4.2](https://rdatatable.gitlab.io/data.table/bin/windows/contrib/4.2/data.table_1.15.99.zip)

**Comment by jangorecki** [comment]
There is documentation inside git repo, not github (wiki)

**Comment by jangorecki** [comment]
your role set to owner

**Comment by ben-schwen** [comment]
Regarding the number of members @ Gitlab: currently we would be 5 including Toby, lets set it to 10?

**Comment by ben-schwen** [comment]
I set the number to 10 ppl. We have to renew the license on 29th July 2025. I also added Toby to the repo.
